Makefile

```diff
@@ -30,8 +31,17 @@ format:
 	black -l 100 $(python-packages)
 	isort $(python-packages)
 
-test:
-	python3 -m pytest -n$(shell nproc) -vv tests
+test: test_unit test_integration
+
+test_unit:
+	python3 -m pytest -vv tests
+
+$(TMPDIR)/bin/pylint:
+	python3 -m venv $(TMPDIR)
+	$(TMPDIR)/bin/pip install pylint .[devel]
+
+test_integration: $(TMPDIR)/bin/pylint
+	$< tests/data/pylint_boolean_argument_invalid.py
```

Comment:
Shouldn't there be a cleanup for the temporary directory?

Reply:
Added.

Reply:
A new temporary directory is created on each invocation of make, even if no temporary directory is required. But the temporary directory is only removed for `test_integration`. Is there a better solution?

Reply:
No, not with `make`. I'm not really comfortable with removing it in the target either, as it may create issues should other targets using the temporary directory be introduced later (in which case the presence of the directory depends on the order of execution). TBH, I don't see a big problem keeping the directory - most systems clean their `/tmp` when rebooting.

Reply:
Remove cleaning of temp directory as discussed separately.